docs/tutorials/developer-journey/level-1/1.1-live-demo.md

```diff
@@ -0,0 +1,143 @@
+# 1.1: Exploring a live demo
+
+## Overview
+
+In the previous segment of our developer journey, introduction to `dfx`, we created a new `dfx` project using the default 'Hello, world' template and looked into the project's structure and code. However, we didn't build or deploy the canister. 
+
+In this tutorial, we'll deploy the `hello_world_backend` canister from [0.6: Introduction to dfx](../level-0/06-intro-dfx.md) to the Motoko playground.
+
+## What is the Motoko playground?
+
+The Motoko playground is a development environment that hosts a 'canister pool' that canisters can be deployed to. When canisters are deployed to the Motoko playground, they temporarily borrow resources from the pool and are allotted a small amount of cycles to can be used during the canister's deployment. Canisters deployed to the Motoko playground can run for 20 minutes; after 20 minutes, the canister will be uninstalled and the resources borrowed are returned to the playground's canister pool. 
```

Comment:
Wouldn't it make sense to call it just "playground" instead of "Motoko playground"? You could mention that this is the background service of the ["Motoko Playground"](https://play.motoko.org/). 

If you want to stay with Motoko Playground, I'd suggest mentioning that this works for canisters developed in other programming languages as well.